packages/terra-action-footer/tests/wdio/theme/block-action-footer-theme-spec.js

```diff
@@ -5,13 +5,13 @@ const {
 
 describe('Block action footer theme', () => {
   describe('No Actions', () => {
-    beforeEach(() => browser.url('/#/raw/tests/terra-action-footer/action-footer/without-actions-action-footer-container'));
+    beforeEach(() => browser.url('/#/raw/tests/terra-action-footer/action-footer/without-actions-block-action-footer'));
 
     Terra.should.themeEachCustomProperty(withoutActionsThemeables);
   });
 
   describe('With Actions', () => {
-    beforeEach(() => browser.url('/#/raw/tests/terra-action-footer/action-footer/with-actions-action-footer-container'));
+    beforeEach(() => browser.url('/#/raw/tests/terra-action-footer/action-footer/with-actions-block-action-footer'));
```

Comment:
These routes were a direct copy paste from the original file, are they also wrong here?

https://github.com/cerner/terra-core/blob/master/packages/terra-action-footer/tests/wdio/block-action-footer-spec.js